test/style.js

```diff
@@ -53,3 +53,10 @@ test('vendor prefixed styles are correctly camel-cased', function (t) {
   t.plan(1)
   t.equal(el.style.WebkitTransition, 'opacity 100ms ease')
 })
+
+test('pascal-cased, vendor prefixed styles are not camel-cased', function (t) {
+  var el = mk().node()
+  el.setAttribute('style', 'WebkitTransition: opacity 100ms ease')
```

Comment:
Ah! I didn't know that, thanks a lot! So this is a valid case, but I think we should handle it by changing the existing if statement I mentioned in my other comment to use a regular expression like `/^[\-A-Z]/`.